src/collective/volto/formsupport/browser/send_mail_template_table.pt

```diff
@@ -2,14 +2,21 @@
   i18n:domain="collective.volto.formsupport"
   define="parameters python:options.get('parameters', {});
             url python:options.get('url', '');
-            title python:options.get('title', '');">
+            title python:options.get('title', '');
+            block python:options.get('block', '');
+            ">
   <style>
     th {
         text-align: start;
     }
   </style>
+  <header>
+    <tal:element tal:replace="structure python:block.get('mail_header', '')"></tal:element>
+  </header>
   <table border="1">
-    <caption i18n:translate="send_mail_text_table">Form submission data for ${title}</caption>
+    <tal:element tal:condition="python:not block.get('mail_header', '')">
```

Comment:
Is this a default when mail_header is not set? if yes, it should be inside the same <header> tag and not here.

Reply:
what do u mean, example?